docs/docs/examples/instrumentation/instrumentation_observability_rundown.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>that's right, extra bit of information if you want to include (totally up to you!): by default, we use the <code>NullSpanHandler</code> and <code>NullEventHandler</code> both of which do nothing :)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>